.travis.yml

```diff
@@ -3,13 +3,22 @@ dist: xenial
 language: cpp
 
 env:
+  global:
+    - SONAR_SCANNER=OFF
+    - COVERALLS=OFF
+    - RUN_TESTS=ON
+    - STOCK_CPPZMQ=ON
+    - USE_PCH=ON
+    - CMAKE_BUILD_TYPE=Debug
+    - TANGO_USE_USING_NAMESPACE=ON
   matrix:
-    - OS_TYPE=gcc-latest CMAKE_BUILD_TYPE=Debug   SONAR_SCANNER=OFF COVERALLS=OFF RUN_TESTS=OFF WARNINGS_AS_ERRORS=ON STOCK_CPPZMQ=ON TANGO_USE_USING_NAMESPACE=OFF USE_PCH=ON
-    - OS_TYPE=debian10 CMAKE_BUILD_TYPE=Debug   SONAR_SCANNER=OFF COVERALLS=OFF RUN_TESTS=ON STOCK_CPPZMQ=ON TANGO_USE_USING_NAMESPACE=OFF USE_PCH=ON
-    - OS_TYPE=debian10 CMAKE_BUILD_TYPE=Release SONAR_SCANNER=OFF COVERALLS=OFF RUN_TESTS=ON STOCK_CPPZMQ=ON
-    - OS_TYPE=debian9  CMAKE_BUILD_TYPE=Debug   SONAR_SCANNER=OFF COVERALLS=OFF RUN_TESTS=ON STOCK_CPPZMQ=ON
-    - OS_TYPE=debian8  CMAKE_BUILD_TYPE=Debug   SONAR_SCANNER=ON  COVERALLS=ON  RUN_TESTS=ON STOCK_CPPZMQ=OFF
-    - OS_TYPE=debian7  CMAKE_BUILD_TYPE=Debug   SONAR_SCANNER=OFF COVERALLS=OFF RUN_TESTS=ON STOCK_CPPZMQ=ON
+    - OS_TYPE=gcc-latest RUN_TESTS=OFF WARNINGS_AS_ERRORS=ON TANGO_USE_USING_NAMESPACE=OFF
+    - OS_TYPE=debian10
+    - OS_TYPE=debian10 USE_PCH=OFF RUN_TESTS=OFF
```

Comment:
We don't run the tests in this case.
We are confident enough the behaviour is (and will stay) the same as when USE_PCH=ON?

Reply:
I'm not against enabling tests here, but the important fact is that we still *compile* the tests. If the behavior is different and the compilation is not failing, then it's rather a compiler issue.

Reply:
Yes I'm confident that just compiling code and tests is enough.